core/domain/user_services.py

```diff
@@ -224,7 +224,9 @@ def _save_user_settings(user_settings):
 
 def has_user_registered_as_editor(user_id):
```

Comment:
Edit: I actually think something like 'is_valid_contributor' might be better, since this is checking more than just a registration status. For whichever name is most appropriate, ditto for all the other similar name recommendations.